ExcelWriter.py

```diff
@@ -19,6 +20,36 @@ def __init__(self, name):
             self.maxCol = []
             self.indentFormat = {}
             self.name=name
+            self.writtenCells = {}
+            
+    def Add_Header(self, headerLines):
+        row = 0;
+        maxCol = 0;
+        for header in headerLines:
+            col = 0
+            for headerLine in header:
+                self.Write_String(row, col, headerLine)
+                col = col + 1
+                maxCol = max(col, maxCol)
+        self.currentCol = maxCol
```

Comment:
The result of this will always be `col`, right? It looks like we could drop `maxCol` altogether.

Reply:
Yeah, this should have been row - in the new upload.  Not sure how to generate a pull request of a new upload (this is not the mode of work I use at Intel (in there we upload the changes to code collaborator & only push when approved) so I'm doddering along with this)

Reply:
Once you've merged a pull request, there isn't really a way to update it. I think the expected work flow is to create a new pull request at that point. Having said that, I don't have any strong feelings about process. Unless you want code review for something, I would say feel free to push directly to master.